about.html

```diff
@@ -0,0 +1,108 @@
+    <script src="https://kit.fontawesome.com/a329405765.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="body-overlay hidden"></div>
+    <header>
+        <section class="user-acc">
+            <div class="social-icon">
+                <i class="fa-brands fa-twitter"></i>
+                <i class="fa-brands fa-facebook"></i>
+            </div>
+            <div class="user">
+                <nav class="nav-user">
+                    <ul>
+                        <li class="logout">Logout</li>
+                        <li class="mypage">My page</li>
+                        <li class="langue">English</li>
+                    </ul>
+                </nav>
+            </div>
+        </section>
+        <section class="main-menu">
+            <a href="./index.html"><img src="./image/logo.png" alt="logo" class="logo" width="45"></a>
+            <nav class="main-nav">
+                <i class="fa-sharp fa-solid fa-bars hamberger"></i>
+                <i class="fa-solid fa-xmark close hidden"></i>
+                <ul class="menu">
+                    <li class="about menu-i"><a href="#">About</a></li>
+                    <li class="class menu-i"><a href="#">Class</a></li>
+                    <li class="join menu-i"><a href="#">Join</a></li>
+                    <li class="teacher menu-i"><a href="#">Teacher</a></li>
+                    <li class="news menu-i"><a href="#">News</a></li>
+                </ul>
+            </nav>
+        </section>
+    </header>
+    <main>
+        <section class="header">
+            <div class="header-overlay"></div>
+            <p class="sub-header">"Hello! Sharing world"</p>
+            <h1>Bim academy training</h1>
+            <p class="description">It feels like the world is moving at a faster pace than ever before. New building regulations, the climate crisis, net zero, ever changing sustainability and safety standards and targets – if any of these are on your mind, BIM Academy can help. Developed and run by BIM Group with over 100 years of built environment knowledge, BIM Academy regularly updates materials to reflect these changes.</p>
+            <div class="contact-info">
+                <p>Please contact us per Email for any further question about BIM Academy!</p>
+                <span class="email">bim-academy@gmail.com</span>
+            </div>
+        </section>
+        <section class="emphasic">
+            <h3>BIM Academy is the training arm of the Institution of Civil Engineers</h3>
+            <hr>
+            <p>Whether you are new or aspiring to the profession, or an experienced practitioner, we offer engineering and construction training courses that satisfy knowledge requirements across the breadth of infrastructure, as well as training on skills such as business and project management.</p>
+            <img src="./image/iso-class.webp" alt="iso bim class">
+        </section>
```

Comment:
- [ ] Kindly make sure this section design sticks as much as possible to the one from the template. E.g. the width & height of image, the width of the block text, paddings and margins, spacing at the left and right,... I can suggest you also give some details about your logo and use the logo as image

| Template Design | Your Design |
| ---------------------- | ----------------- |
| ![Screenshot from 2023-01-05 19-56-31](https://user-images.githubusercontent.com/83042742/210849551-07f7b24f-745c-484f-b604-2aed4e6e8711.png) | ![Screenshot from 2023-01-05 20-02-19](https://user-images.githubusercontent.com/83042742/210849578-4ca72958-b4f7-45fa-81be-824f0e82c0f4.png) |